### ISSUE TITLE
Is this repo compatible with tensorflow 1.0?

### ISSUE THREAD

**Issue by crci** [openai/universe-starter-agent#83]
I have been using it with TF 1.0, but I just saw that the documentation says we should use tensorflow 0.12. Should we? Or the docs have not been updated?

Thanks for this great resource!

**Comment by Voidminded** [comment]
Yes it is, **TF 1.x** is using same syntax as **TF 0.12** for which this code is compatible.

**Comment by andrewcby** [comment]
There are some issues with training using 1.0 - it simply just doesn't train

**Comment by KaixiangLin** [comment]
I also observed that all processes stuck with current version in tf1.01. 

I modified one of the previous version so that it can run with tensorflow 1.01. 

Code attached here.  [tf1.01.zip](https://github.com/openai/universe-starter-agent/files/924051/tf1.01.zip)

Should be able to run by typing this command: 
```python train.py --num-workers 8 --env-id PongDeterministic-v3```

**Comment by crci** [comment]
@KaixiangLin thank you. What was the issue with the current code? By any chance have you to trained the A3C in the Neonracer with that code? If so, what score you get in how much time?

@andrewcby with TF 1.0.1 and 6 workers, I managed to get to barely 60K points in neonracer in ~10days. So it learned, but way slower than the Readme says. There is some conversation about this in #52 . I'll try with 0.12 just in case.

**Comment by KaixiangLin** [comment]
@crci  It took around 40 minus for Pong to get 20 rewards. It may be faster if I build tf1.01 from source. Haven't try Neonracer yet.

**Comment by andrewcby** [comment]
After updating to TF 1.1.0, with 16 workers I got the reward 21 within 30 mins like in README, but 32 workers JUST WONT LEARN. I think there might be some issue with compatibility still. 

At the same time, @KaixiangLin I'm also curious what you changed in that code? Since now there's TF 1.1 I wonder if it applies to 1.1 too?

**Comment by KaixiangLin** [comment]
I changed by this 
```python tf_upgrade.py --infile InputFile --outfile OutputFile```
and manually change
```tf.nn.rnn_cell.BasicLSTMCell(size, state_is_tuple=True)``` to  ```tf.contrib.rnn.BasicLSTMCell(size, state_is_tuple=True)```
Those changes are not made to current version of starter agent, it's based on the previous version of it (around two month ago when the code is only using tf 0.11)

**Comment by GokulNC** [comment]
Is this repo compatible with the latest Tensorflow 1.5.0??
